vaadin-side-nav-flow-parent/vaadin-side-nav-flow/src/main/java/com/vaadin/flow/component/sidenav/SideNavItem.java

```diff
@@ -221,6 +232,114 @@ public String getPath() {
         return getElement().getAttribute("path");
     }
 
+    /**
+     * Adds the specified path aliases to this item. The aliases act as
+     * secondary paths when determining the active state of the item.
+     *
+     * @param aliases
+     *            the path aliases to add to this item
+     */
+    public void addPathAlias(String... aliases) {
```

Comment:
I wonder if the method should be named `addPathAliases` given that it supports adding multiple aliases at once.

Reply:
Updated the names.